rosidl_generator_cpp/cmake/rosidl_generator_cpp_generate_interfaces.cmake

```diff
@@ -106,6 +106,7 @@ add_dependencies(
 
 set(_target_suffix "__rosidl_generator_cpp")
 add_library(${rosidl_generate_interfaces_TARGET}${_target_suffix} INTERFACE)
+target_compile_features(${rosidl_generate_interfaces_TARGET}${_target_suffix} INTERFACE cxx_std_17)
```

Comment:
Calling this out - is it a concern? We've upgraded everything else to C++17 but the generated interface libraries themselves hadn't actually specified a standard version. This is  needed for the `static constexpr rosidl_type_hash_t` in non-templated `Service` and `Action` classes, for the inline variable thing that's new in C++17

Reply:
I don't think this is a concern; we've specified in REP-2000 that we target C++17, and we've upgraded most of the packages at this point to C++17.  So I'm fine with adding this here.

Reply:
Fine with me too.